.github/workflows/ci.yml

```diff
@@ -39,4 +39,10 @@ jobs:
 
     - name: Run HTML5 Validator
       run: html5validator --root _site/
+    
+    - name: Serve with jekyll
+      run: bundle exec jekyll serve
```

Comment:
This one won't exit (so it ran 6 hours before I cancelled it). Probably need to join it with the next job and figure out how to run it in the background